app/views/static_pages/about.html.erb

```diff
@@ -0,0 +1,11 @@
+<% provide(:title, "About") %>
+<h1><%= t ".title" %></h1>
+<p>
+  The <a href="https://www.railstutorial.org/"><em>Ruby on Rails
```

Comment:
I18n cho toàn bộ text ở toàn bộ html chú nhé, ko làm đại diện thế này

ngoài ra, dùng `link_to` thay vì `a` tag nhé